cmsmon-tools.spec

```diff
@@ -60,6 +61,14 @@ chmod +x stern_linux_amd64
 cp stern_linux_amd64 %i/stern
 cd -
 
+# install token-manager
+cd %{_builddir}
+curl -ksLO https://github.com/vkuznet/auth-proxy-server/releases/download/%apsver/auth-proxy-tools.tar.gz
```

Comment:
@vkuznet , why not add this to sources?

Reply:
Shahzad, honestly I don't remember RPM spec command syntax, and I find it very difficult to find the appropriate docs for them. The setup syntax I obviously don't remember, e.g. how it untar/unzip all possible combinations, and it is much easier to call curl instead of using sources/setup. If you remember proper syntax feel free to let me know how to do it (I can't do man rpm synax, and googling rpm setup does not lead anywhere close to how RPM commands are structured and which options to use for different use-cases).

Reply:
I mean you already have 4 sources defined here https://github.com/cms-sw/cmsdist/pull/6731/files#diff-15398a788af5b8c0e165e8cca9a44fb7637daebb4850bf66571da9512729a925L11-L14 , so why not just and this as a `Source4` and extract is just like you extra other sources https://github.com/cms-sw/cmsdist/pull/6731/files#diff-15398a788af5b8c0e165e8cca9a44fb7637daebb4850bf66571da9512729a925L24

Reply:
by the way the RPM doc here https://rpm-packaging-guide.github.io/ looks good. See the `%setup` section for details how to expand